easybuild/easyblocks/c/clang.py

```diff
@@ -84,16 +94,26 @@ def __init__(self, *args, **kwargs):
         self.llvm_obj_dir_stage3 = None
         self.make_parallel_opts = ""
 
-        unknown_targets = [target for target in self.cfg['build_targets'] if target not in CLANG_TARGETS]
+        build_targets = self.cfg['build_targets']
```

Comment:
It hasn't been renamed; `unknown_targets` is still there (see last line of the inserted block).  It just looks strange in the diff. `build_targets` is a newly introduced variable that is used later on.